scripts/gke/deployment_manager_configs/cluster.jinja

```diff
@@ -66,20 +66,10 @@ resources:
       # Create a very small minimal pool. Actual nodes will be managed
       # as additional node pools. This makes it easier to
       initialNodeCount: {{ properties['cpu-pool-initialNodeCount'] }}
-      {% if properties['stackdriver-kubernetes'] %}
-      # TODO: remove alpha when 10.2 is public.
-      # https://github.com/kubeflow/kubeflow/issues/821
-      enableKubernetesAlpha: true
       # We need 1.10.2 to support Stackdrivier GKE.
       initialClusterVersion: 1.10.2-gke.0
```

Comment:
Now most recent version is 1.10.5-gke.3, and 1.10.2-gke.0 is no long available from UI.